src/components/Form/index.js

```diff
@@ -1,12 +1,12 @@
 import DocumentSelection from './DocumentSelection';
 import FormSection from './FormSection';
-import SetAclPermissionForm from './SetAclPermissionForm';
-import SetAclPermsDocContainerForm from './SetAclPermsDocContainerForm';
+import SetAclPermissionModal from '../Modals/SetAclPermissionModal';
```

Comment:
Since they're already inside `src/components/Modals` why not have them imported from the `index.js` file there?

Reply:
OK, I'll get rid of them here and leave them in the Modal folder one. I'm not familiar with importing components into these different `index.js` files. Could you give me the proper term to search so I can learn more about why/how to do it?

Reply:
I'm not too sure about the name for this, but basically `index.js` acts as a relay for existing exports in the existing directory and allows it to export them as named imports. It let's us import the path for the directory other than the path for the file.